modules/bound-sa-tokens-configuring.adoc

```diff
@@ -19,11 +19,9 @@ You can configure pods to request bound service account tokens by using volume p
 +
 This step is typically not required if the bound tokens are used only within the cluster.
 +
-[WARNING]
+[IMPORTANT]
 ====
-If you update the `serviceAccountIssuer` field and there are bound tokens already in use, all bound tokens with the previous issuer value will be invalidated. Unless the holder of a bound token has explicit support for a change in issuer, the holder will not request a new bound token until pods have been restarted.
-
-If necessary, you can manually restart all pods in the cluster so that the holder will request a new bound token. Before doing this, wait for a new revision of the Kubernetes API server pods to roll out with your service account issuer changes.
```

Comment:
> If necessary, you can manually restart all pods in the cluster so that the holder will request a new bound token. Before doing this, wait for a new revision of the Kubernetes API server pods to roll out with your service account issuer changes.

I would keep this section here to ensure we indicate that workloads that use bound tokens might need to restart to pick up a new bound token. 

/cc @stlaz

Reply:
@mfojtik I thought you had said that we didn't need the steps to manually restart all pods? Are you saying you want me to keep this statement in the note, and also bring back those steps in the procedure?

Reply:
@bergerhoffer "if necessary" is the culprit here... we want to make sure that if somebody runs a workload that uses a bound token for something, that workload is aware the service account changed. If the workload is smart, it always reads it from the disk in a container (and it will be updated there)... if the workload is not smart, they read it into memory and it might need a restart to update.

with this sentence, I want to ensure that we cover the "not so-smart" workloads.

Reply:
Gotcha. Sentence + steps put back as requested :)